docs/updating/migration-to-cdn/angular.md

```diff
@@ -0,0 +1,180 @@
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
+import { SlashCommand } from 'ckeditor5-premium-features';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor = ClassicEditor;
+	public config = {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+		plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+		toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+		// mention: {
+		//     Mention configuration
+		// }
+	}
+}
+```
+
+**After:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor;
+	public config;
+
+	public ngOnInit() {
+		loadCKEditorCloud( {
+			version: '43.0.0'
+		} )
+			.then( this._setupEditor.bind( this ) );
+	}
+
+	private _setupEditor( cloud ) {
+		const {
+			ClassicEditor, Bold, Essentials, Italic,
+			Mention, Paragraph, SlashCommand, Undo
+		} = cloud.CKEditor;
+
+		this.Editor = ClassicEditor;
+		this.config = {
+			licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+			plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+			toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+			// mention: {
+			//     Mention configuration
+			// }
+		}
+	}
+}
+```
+
+```html
+// app.component.html
+
+<ckeditor
+	*ngIf="Editor"
+	[editor]="Editor"
+	[config]="config"
+	...
+</ckeditor>
+```
+
+### Step 3 (Optional): Migrate the CKEditor&nbsp;5 Angular integration testing suite
+
+If you have any tests that use CKEditor&nbsp;5 objects, you need to update them to use the  `loadCKEditorCloud` function. Here is an example of migrating a test that uses the `ClassicEditor` object:
+
+**Before:**
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+
+it( 'ClassicEditor test', () => {
+	// Your test that uses the CKEditor 5 object.
+} );
+```
+
+**After:**
+
+```javascript
+import { loadCKEditorCloud } from '@ckeditor/ckeditor5-integrations-common';
+
+let cloud;
+
+beforeEach( async () => {
+	cloud = await loadCKEditorCloud( {
+		version: '{@var ckeditor5-version}',
+	} );
+} );
+
+it( 'ClassicEditor test', () => {
+	const { ClassicEditor, ... } = cloud.CKEditor;
+
+	// Your test that uses the CKEditor 5 object.
+} );
+```
+
+### Step 4 (Optional): Clean up the head entries before each test
```

Comment:
`head` is very general term, and for 5s I had a wtf that this means. Either use `<head>` or some more descriptive form.